src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -89,13 +99,34 @@ public function storeOperatesOnASingleChannel()
 
     /**
      * @Given /^the store operates on (?:a|another) channel named "([^"]+)"$/
+     * @Given the store operates on a channel identified by :code code
      */
-    public function theStoreOperatesOnAChannelNamed($channelName)
+    public function theStoreOperatesOnAChannelNamed($channelIdentifier)
     {
-        $channel = $this->channelFactory->createNamed($channelName);
-        $channel->setCode($channelName);
+        $channel = $this->channelFactory->createNamed($channelIdentifier);
+        $channel->setCode($channelIdentifier);
 
         $this->channelRepository->add($channel);
         $this->sharedStorage->set('channel', $channel);
     }
+
+    /**
+     * @Given the channel :channel is enabled
+     */
+    public function theChannelIsEnabled(ChannelInterface $channel)
+    {
+        $channel->enable();
+        $this->channelManager->flush();
+        $this->sharedStorage->set('channel', $channel);
+    }
+
+    /**
+     * @Given the channel :channel is disabled
```

Comment:
I'm not convinced. We could do it in one function, but we would have to add if statement. I think it is more readable this way. When we can check condition a little bit earlier. 
Going further that way, I could us `(enabled|disabled)` as a string parameter `state`, cut off last letter and just call `$channel-$state()`. PHP magic...
Maybe extracting 
```
        $this->channelManager->flush();
        $this->sharedStorage->set('channel', $channel);
```
would be a good solution?